File: javascript/Shubham Sigdar/Sum Mixed Array.js
// Given an array of integers as strings and numbers, return the sum of the array values as if all were numbers.

// Return your answer as a number.

//P:array of integers as strings and numbers
//R: return the sum of the array values Return your answer as a number
//E:
//P: arr.reduce((sum, current) => sum + current, 0);
function sumMix(x){
    let result= x.reduce((sum,current)=>sum+Number(current),0)
     return result;
   }
   console.log(sumMix([9, 3, '7', '3']), 22)
   console.log(sumMix(['5', '0', 9, 3, 2, 1, '9', 6, 7]), 42)
   console.log(sumMix(['3', 6, 6, 0, '5', 8, 5, '6', 2,'0']), 41)

File: java/Shubham Sigdar/SumMixedArray.java
import java.util.List;
//Given an array of integers as strings and numbers, return the sum of the array values as if all were numbers.
//
//        Return your answer as a number.

public class SumMixedArray {
    public int sum( List<?> mixed) {
        int sum=0;
        for(Object i:mixed){
            sum+=Integer.parseInt(i.toString());
        }
        return sum;
    }
}
